index.js

```diff
@@ -313,9 +412,10 @@ alexa.app = function(name, endpoint) {
         }
       };
       try {
+        var context = request.context();
         var requestType = request.type();
         if (typeof self.pre == "function") {
-          self.pre(request, response, requestType);
+          self.pre(request, response, requestType, context);
```

Comment:
I think `pre` shouldn't take the 4th parameter since `request.context` is a thing. You can access that inside `pre`.